### PR TITLE
isort cr.cube

### PR DESCRIPTION
**Pull request by slobodan-ilic**

### CODE REVIEW

**Review by ernestoarbitrio — APPROVED** [review]
Awesome 👏🏻

File: src/cr/cube/cube.py [hunk 1]
```diff
@@ -5,7 +5,12 @@
 CubeSet is the main API class for manipulating Crunch.io JSON cube responses.
 """
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
```
Comment (scanny) [inline src/cr/cube/cube.py:13]: This line was fine as it was. `isort` needs `-l 88` or `--line-length 88` to wrap at the same place as Black.

File: src/cr/cube/cube.py [hunk 2]
```diff
@@ -14,7 +19,9 @@
 
 from cr.cube.cubepart import CubePartition
 from cr.cube.dimension import AllDimensions
-from cr.cube.enums import CUBE_MEASURE, DIMENSION_TYPE as DT, NUMERIC_MEASURES
+from cr.cube.enums import CUBE_MEASURE
+from cr.cube.enums import DIMENSION_TYPE as DT
+from cr.cube.enums import NUMERIC_MEASURES
```
Comment (scanny) [inline src/cr/cube/cube.py:24]: This one was also fine as it was. Repeating the `from x.y.module import ...` prefix obscures the boundaries between source modules in the import list. Adding `--multi-line 8` will change it to the format we already use.

**Review by scanny — CHANGES_REQUESTED** [review]
Okay, now that I've seen all the fixes this made I feel a little better about it. The two configuration changes I've identified are must-haves, there are several unneeded changes when those are left out.

The only thing I still don't like is repeating the `from x.y.module import ...` prefix on imports that rename the imported item, like `from enums import MEASURES as M`. Maybe one of the configuration options can get rid of that.

File: tests/integration/test_cube.py [hunk 3]
```diff
@@ -14,14 +14,12 @@
...
 from cr.cube.enums import DIMENSION_TYPE as DT
 
-from ..fixtures import (
-    CR,  # cube-response
-    OL,  # overlaps
-    NA,  # numeric-array
-)
+from ..fixtures import CR  # cube-response
+from ..fixtures import NA  # numeric-array
+from ..fixtures import OL  # overlaps
```
Comment (scanny) [inline tests/integration/test_cube.py:22]: These comments should go above so these don't break onto separate redundant lines. Or maybe we just get rid of the comments altogether.

**Review by scanny — COMMENTED** [review]
Just this one change I see, otherwise these look real good, that's a lot of fixes it caught :)

**Comment by ernestoarbitrio** [comment]
@slobodan-ilic I'm gonna close this PR and open (soon) a new one with `isort` latest version.

**Pull request closed without merging**